sdk/tests/deqp/framework/common/tcuSkipList.js

```diff
@@ -96,10 +97,28 @@ goog.scope(function() {
     tcuSkipList.getSkipStatus = function(testName) {
         var skipEntry = _skipEntries[testName];
         if (skipEntry === undefined) {
-            return { 'skip': false, 'reason': '' };
+            return this.getSkipsStatus(testName);
         } else {
             return { 'skip': true, 'reason': skipEntry };
         }
     }
 
+    /*
+     * Gets the skip status of the given tests like testpath*
+     * object with the properties "skip", a boolean, and "reason", a
+     * string.
+    */
+    tcuSkipList.getSkipsStatus = function(testName) {
+        var skipEntry;
+        for (var key in _skipEntries) {
+            if (testName.indexOf(key) >=0 ) {
```

Comment:
The native hw-issues.txt and driver-issues.txt in deqp in aosp also use * to skip a bunch of test cases. I will let Yizhou have more investigations about this. Thanks for your feedbacks, @zhenyao and @kenrussell .